ios/FluentUI/SegmentedControl/SegmentedControl.swift

```diff
@@ -194,6 +203,13 @@ open class SegmentedControl: UIControl {
         view.layer.cornerCurve = .continuous
         return view
     }()
+    private let pillMaskedLabelLayerView: UIView = {
```

Comment:
something about this name is confusing. it makes it sound like a labellayer.... but this controls maskview is selectionview?